app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/components/ImportedFilesCard.kt

```diff
@@ -58,20 +58,19 @@ fun ImportedFilesCard(
     modifier: Modifier = Modifier,
     files: () -> List<FileUi>,
     pickFiles: () -> Unit,
-    navigateToFileDetails: () -> Unit,
+    navigateToFilesDetails: () -> Unit,
 ) {
 
     val context = LocalContext.current
-
-    val humanReadableSize by remember {
+    val humanReadableSize by remember(files()) {
```

Comment:
Was this remember key really needed? You're passing down a state all the way down here so the derivedState should handle it be itself